website/static/js/pages/user-addon-cfg-page.js

```diff
@@ -49,3 +50,12 @@ $('#selectAddonsForm').on('submit', function() {
     }
     return false;
 });
+
+addonEnabledSettings = window.contextVars.addonEnabledSettings;
+for (var i=0; i < addonEnabledSettings.length; i++) {
+       var addonName = addonEnabledSettings[i];
+       if (typeof window.contextVars.addonsWithNodes !== "undefined" && addonName in window.contextVars.addonsWithNodes) {
```

Comment:
As per our `.jshintrc`, use singlequotes for strings. Will make this change when I merge.